airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/common/regression_test.py

```diff
@@ -0,0 +1,56 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+from __future__ import annotations
+
+import dagger
+from pipelines.airbyte_ci.connectors.context import ConnectorContext
+from pipelines.models.steps import Step, StepResult, StepStatus
+
+
+class RegressionTest(Step):
```

Comment:
@bleonard I saw this and thought wait, why is live testing here? Then I realized it's not that. Should we rename this to `SpecTest` or `SpecRegressionTest` to be explicit?

Reply:
semantically, I wanted to leave this open to whatever we think we should test and not just be `Spec` -- the pipelines adding it shouldn't care about that detail. Open to other names if you think it's important.